web/src/components/shared/Toast.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+export interface ToastProps {
+  isToastVisible: boolean;
+  type: "success" | "error" | "warning";
+  children: React.ReactNode;
+}
+
+const Toast = ({ isToastVisible, type, children }: ToastProps) => {
+  const toastType = () => {
+    switch (type) {
+      case "success":
+        return "tw-bg-[#38cc97]";
+      case "error":
+        return "tw-bg-[#f65c5c]";
+      case "warning":
+        return "tw-bg-[#FFA500]";
+      default:
+        return "tw-bg-[#38cc97]";
+    }
+  };
+
```

Comment:
```suggestion
const toastType = () => {
  switch (type) {
    case "success":
      return "tw-bg-[#38cc97]";
    case "error":
      return "tw-bg-[#f65c5c]";
    case "warning":
      return "tw-bg-[#FFA500]";
    default:
      return "tw-bg-[#38cc97]";
  }
};
const Toast = ({ isToastVisible, type, children }: ToastProps) => {
```

this should define the function when the module is loaded rather than every time this is rendered. only a suggestion since it probably has no real user impact

Reply:
@rianfowler good point! I'll update it. thanks for pointing it out!